chef.gemspec

```diff
@@ -65,7 +65,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency "aws-sdk-s3", "~> 1.91" # s3 recipe-url support
   s.add_dependency "aws-sdk-secretsmanager", "~> 1.46"
-  s.add_dependency "vault", "~> 0.16" # hashi vault official client gem
+  s.add_dependency "vault", "~> 0.18" # hashi vault official client gem
```

Comment:
Do we want to leave the minor version as the variable for upgrades, or do we want to actually specify `~> 0.18.2` so that the pessimistic version specifier prevents a `bundle update vault` from pushing us to `0.19.x` or beyond?

Reply:
agreed. Updating